### PR TITLE
feat(aapd-1663): move web comment into forms-web-app

### PR DESCRIPTION
**Pull request by bryn-pins**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-1663

## Description of change

Moves web comment into forms-web-app under /comment-planning-appeal
## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/Planning-Inspectorate/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/common/src/regex.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// https://github.com/ideal-postcodes/postcode MIT
+
+/**
+ * Tests for a valid postcode
+ */
+module.exports.fullPostcodeRegex = /^[a-z]{1,2}\d[a-z\d]?\s*\d[a-z]{2}$/i;
```
Comment (bryn-pins) [inline packages/common/src/regex.js:6]: we need to handle ignoring whitespace in postcodes with the sql query at some point

File: packages/web-comment/package.json
Comment (beejjacobs) [inline packages/web-comment/package.json:1]: If we're removing this package there should be some `package-lock` changes too.

File: packages/common/src/regex.js (hunk 1 above)
Comment (topickering) [reply packages/common/src/regex.js:6]: Do we have a bug / feat ticket for this?

Comment (bryn-pins) [reply packages/common/src/regex.js]: I've reverted the postcode regex to enforce spaces for now and will get a ticket in

**Review by topickering — DISMISSED** [review]
Looks good to me, good luck with conflicts!

**Review by topickering — APPROVED** [review]

**Pull request merged**